app/org/sagebionetworks/bridge/play/controllers/ParticipantController.java

```diff
@@ -124,15 +124,13 @@ private Result getParticipantsInternal(Study study, String offsetByString, Strin
         return okResult(page);
     }
     
-    public Result createParticipant(String verifyEmailString) throws Exception {
+    public Result createParticipant() throws Exception {
```

Comment:
Discussed offline. This vulnerability is an existential risk to our platform and needs to be closed. We can improve the developer onboarding experience at a later date. Will file a separate JIRA.